src/Forms/AssessmentSettingsForm.cs

```diff
@@ -361,5 +362,17 @@ private void VisitLink(string goLinkUrl)
             Process.Start(processDescription);
         }
         #endregion
+
+        #region Utilities
+        public void DisableOptimizationPreferenceComboBox()
+        {
+            OptimizationPreferenceComboBox.Enabled = false;
+        }
+
+        public void EnableOptimizationPreferenceComboBox()
+        {
+            OptimizationPreferenceComboBox.Enabled = true;
+        }

```

Comment:
When this is disabled update the selected item in the combo box, so that you don't need to hard-code getting it based on the proposal type.